service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -108,30 +103,25 @@ public void notifyAdmin(final AdditionalApplicationsUploadedEvent event) {
 
             NotifyData notifyData = contentProvider.getNotifyData(caseData);
             String recipient = courtService.getCourtEmail(caseData);
-            String template = featureToggleService.isServeOrdersAndDocsToOthersEnabled()
-                              ? UPDATED_INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE_CTSC
-                              : INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE;
-
-            notificationService.sendEmail(template, recipient, notifyData, caseData.getId());
+            notificationService.sendEmail(
+                UPDATED_INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE_CTSC, recipient, notifyData, caseData.getId());
```

Comment:
would it be worth updating this template name now that the old one is removed?

Reply:
it seems that there is another that is defined as `INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE_CTSC` any reason for that?

Reply:
looks like it is identical to the one that you removed, will remove that one as well and update the name of the current one